native_client/python/client.py

```diff
@@ -35,6 +38,13 @@
 # Size of the context window used for producing timesteps in the input vector
 N_CONTEXT = 9
 
+def convert_samplerate(audio_path):
+    temp_dir = tempfile.mkdtemp()
+    output_path = os.path.join(temp_dir, 'output.wav')
+    sox_cmd = 'sox --norm {} -b 16 {} channels 1 rate 16000'.format(audio_path, output_path)
+    subprocess.Popen(sox_cmd, shell=True).wait()
```

Comment:
This is unsafe, using `shell=True` requires escaping the user input with `shlex.quote`. Is `shell=True` required here?

I would also like some error handling. If the user doesn't have sox installed, for example, we should show an error message instead of terminating with an exception.